ui/apps/search/src/components/routes/search-page.tsx

```diff
@@ -288,7 +288,10 @@ const SearchPage: React.FC<SearchPageProps> = props => {
       showLoginModal();
       return;
     } else {
-      props.navigateToModal({ name: 'editor-modal', embedEntry: entryId });
+      navigateTo?.({
+        appName: '@akashaorg/app-akasha-integration',
+        getNavigationUrl: () => `post/${entryId}?action=repost`,
```

Comment:
you can use the same approach as in the `ui/lib/feed/src/component/entry-feed.tsx` -> `getNavigationUrl: navRoutes => `${navRoutes.Post}/${entryId}?action=repost`,`